Sources/Server/Response.swift

```diff
@@ -3,55 +3,80 @@ import Foundation
 public class Response {
 
     let parser: Parser
-    var paths: Array<String>
+    var paths: [String: (Response) -> (Request) -> (String)] = ["/": handleRoot,"/hello": handleHello,"/coffee": handleCoffee, "/tea": handle200]
```

Comment:
Any reason why you can't or shouldn't reuse handleRoot for handle200?

Reply:
I thought about that, but handleRoot has "Hello World" in the body, and the requirements for "/tea" are an empty body that send back a 200 status code. Should I think about removing "Hello World"?

Reply:
That's up to you! I do think the Hello World on the root is a nice indicator that you have the server running!